src/com/hillel/course2/lesson6/homework/Main.java

```diff
@@ -131,4 +102,25 @@ private static boolean checkNewStatus(OrderStatusEnum newStatus, OrderStatusEnum
                 return false;
         }
     }
+
+    private static void serialization(Map<Integer, Order> map) {
+        try (FileOutputStream fileOutputStream = new FileOutputStream(FILE_PATH)) {
+            ObjectOutputStream objectOutputStream = new ObjectOutputStream(fileOutputStream);
+            objectOutputStream.writeObject(map);
+            objectOutputStream.flush();
```

Comment:
you can use try with resources for multiple resources as well https://stackoverflow.com/questions/47175526/try-with-multiple-resource-in-java
In this case you don't need to close the second resource by yourself